partner_contact_personal_information_page/README.rst

```diff
@@ -42,6 +65,7 @@ Contributors
 * Jairo Llopis <j.llopis@grupoesoc.es>
 * Matjaž Mozetič <m.mozetic@matmoz.si>
 * Rudolf Schnapka <schnapkar@golive-saar.de>
+* Richard deMeester <richard@willowit.com.au>
 
 Maintainer
```

Comment:
Please check the part Maintainer in https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst